README.md

```diff
@@ -19,6 +19,7 @@ Rust (user-space) virtual machine for eBPF
 * [Building eBPF programs](#building-ebpf-programs)
 * [Feedback welcome!](#feedback-welcome)
 * [Questions / Answers](#questions--answers)
+* [No_std Support](#no_std-support)
```

Comment:
Let's maybe have a “Build features” section (or something like this) instead, and move it higher up in the list (above “Feedback welcome!” maybe). Then we can have `no_std` as a subsection of that? We need to document the Cranelift JIT at some point (not in this PR of course), and it would be another subsection of “Build features”.